src/ffi.rs

```diff
@@ -77,7 +77,8 @@ impl PublicKey {
     /// Create a new (zeroed) public key usable for the FFI interface
     pub fn new() -> PublicKey { PublicKey([0; 64]) }
     /// Create a new (uninitialized) public key usable for the FFI interface
-    pub unsafe fn blank() -> PublicKey { mem::uninitialized() }
+    #[deprecated(since = "0.15.2", note = "Please use the new function instead")]
+    pub unsafe fn blank() -> PublicKey { PublicKey::new() }
```

Comment:
Both `new` and `blank` are really weird. I had completely forgotten it existed. Can you trigger assertation failures with the output of `PublicKey::new`?

I think we should deprecate both.

Reply:
We couple implement `Default` and have it return the (1/2)G pubkey, which would be a very visible sentinel. Or the `02`x33 pubkey? I think users do want a way to get a default/uninitialized value. But giving them one that is not actually a valid `Pubkey` does not seem legitimate.

Reply:
Yeah it's a bit weird to give a new/default with zeros. though remember that this is the ffi pubkey. not the one we give as the API.

Reply:
At a second PR I could refactor to remove the manual implementation of the clone/copy/equal etc. with the derived ones if you prefer (on one hand I prefer the derived ones on the other it might be a breaking change that isn't entirely visible)

Reply:
Oh, that's right, I always confuse `ffi::PublicKey` with the real `PublicKey`.

Yeah, `new` is fine. I guess we should also implement `Default` in terms of it.

Reply:
`Default` is actually already implemented using `new`.